packages/shared/lib/services/connections/utils.ts

```diff
@@ -6,7 +6,7 @@ export const DEFAULT_EXPIRES_AT_MS = ms('1day');
 export const DEFAULT_OAUTHCC_EXPIRES_AT_MS = ms('55minutes'); // This ensures we have an expiresAt value
 export const DEFAULT_BILL_EXPIRES_AT_MS = ms('35minutes'); //This ensures we have an expireAt value for Bill
 export const DEFAULT_INFINITE_EXPIRES_AT_MS = ms('99years');
-export const MAX_FAILED_REFRESH = 30;
+export const MAX_FAILED_REFRESH = 3;
```

Comment:
see my comment above. If we want to ensure at least 3 days we should have 4 here to make sure 3 full days have elapsed. The first one might have been partial

Reply:
I would also rename this env var and even maybe the column in the db since it doesn't represent the number of failure anymore but the number of days it has been failing

Reply:
renaming a column used in prod is a bit dangerous tbh, but I'll rename the env var for sure 👌🏻